src/components/Shared/Window.js

```diff
@@ -63,121 +64,123 @@ const Window = React.memo(
     }, [expanded])
 
     return (
-      <Box
-        id='parent'
-        sx={{
-          bottom: 0,
-          position: 'absolute',
-          width: spacing ? containerWidth : '100%',
-          height: spacing ? containerHeight : '100%',
-          backgroundColor: 'rgba(0, 0, 0, 0.1)',
-          display: 'flex',
-          justifyContent: 'center',
-          alignItems: 'center'
-        }}
-      >
-        <Draggable
-          handle='#draggable-dialog-title'
-          cancel={'[class*="MuiDialogContent-root"]'}
-          bounds='parent'
-          position={expanded && { x: 0, y: 0 }}
-          onStart={() => draggable}
+      <CacheDataProvider>
```

Comment:
should the CacheDataProvider be here or abothe the windowProvider in the app.js

Reply:
Hadi move it from the app to here